src/messages/Image.cpp

```diff
@@ -495,6 +495,20 @@ int Image::height() const
     return 16;
 }
 
+QSize Image::size() const
+{
+    assertInGuiThread();
+
+    if (auto pixmap = this->frames_->first())
+    {
+        return pixmap->size() * this->scale_;
+    }
+
+    // TODO: Some images contain size as part of their API, we use could use that instead of
```

Comment:
```suggestion
    // TODO: Some images contain size as part of their API, we could use that instead of
```

This seems like it is a duplicate word `use`